ui/modal/info_modal.go

```diff
@@ -50,7 +50,7 @@ func NewInfoModal(l *load.Load) *InfoModal {
 		btnPositve:   l.Theme.OutlineButton("Yes"),
 		btnNegative:  l.Theme.OutlineButton("No"),
 		keyEvent:     l.Receiver.KeyEvents,
-		isCancelable: true,
+		isCancelable: false,
```

Comment:
Revert this.
due to this, the below modal cannot be minimized by clicking the background.
<img width="800" alt="image" src="https://user-images.githubusercontent.com/27733432/141420172-f2f823e8-cc48-4816-9dc6-76ef0add8465.png">